packages/feature-layer/src/feature-layer.ts

```diff
@@ -333,6 +333,7 @@ export function postProcess(
   itemId: string,
   type: string,
   templates: common.IItemTemplate[],
+  template: common.IItemTemplate,
```

Comment:
Fixes a regression that was introduced on Friday in [this commit](https://github.com/Esri/solution.js/commit/76bec4ab071a8edf706c31d83576410e1a3ecf65#diff-266ab901f19c8c388edb8eb5976c2c7b)